.github/workflows/performance.yml

```diff
@@ -123,5 +123,18 @@ jobs:
       - name: Print Performance tests results
         run: "node ./tests/performance/results.js"
 
+      - uses: actions/github-script@98814c53be79b1d30f795b907e553d8679345975 # v6.4.0
+        id: commit-timestamp
+        with:
+            github-token: ${{secrets.GITHUB_TOKEN}}
+            script: |
+                const commit_details = await github.rest.git.getCommit({owner: context.repo.owner, repo: context.repo.repo, commit_sha: context.sha});
+                return parseInt((new Date( commit_details.data.author.date ).getTime() / 1000).toFixed(0))
+      - name: Publish performance results
+        env:
+            COMMITTED_AT: ${{ steps.commit-timestamp.outputs.result }}
+            CODEHEALTH_PROJECT_TOKEN: ${{ secrets.CODEHEALTH_PROJECT_TOKEN }}
+        run: node ./tests/performance/log-results.js $CODEHEALTH_PROJECT_TOKEN trunk $GITHUB_SHA debd225d007f4e441ceec80fbd6fa96653f94737 $COMMITTED_AT
```

Comment:
Here you are passing 5 arguments: token, branch, hash, baseHash, and timestamp, but in the `log-results.js` file you are not accounting for the `baseHash`. You can probably remove the baseHash value here. Also note the suggestion I made in the `log-results.js` file about passing the hostname for the API as another CLI argument here.